llumnix/entrypoints/vllm/api_server.py

```diff
@@ -214,6 +215,10 @@ async def is_ready():
     ready_status = await engine_manager.is_ready.remote()
     return ready_status
 
+def set_runtime_env(engine_manager_args: EngineManagerArgs):
+    os.environ['MIGRATE_CACHE_SIZE'] = str(engine_manager_args.migration_cache_blocks)
```

Comment:
This looks strange. Why do we need pass parameters using envvars *inside our own program*?

Reply:
I will revise it.